linter/linter.go

```diff
@@ -268,7 +272,11 @@ func (l *Linter) lintVCL(vcl *ast.VCL, ctx *context.Context) types.Type {
 
 	// Lint each statement/declaration logics
 	for _, s := range statements {
-		l.lint(s, ctx)
+		func(v ast.Statement, c *context.Context) {
```

Comment:
should we define this as a helper function?

Reply:
You are right, will do 👍